src/opnsense/mvc/app/models/OPNsense/Diagnostics/Netflow.xml

```diff
@@ -27,6 +27,7 @@
             <targets type="CSVListField">
                 <Mask>/^((?:(?:25[0-5]|2[0-4][0-9]|[01]?[0-9][0-9]?)\.){3}(?:25[0-5]|2[0-4][0-9]|[01]?[0-9][0-9]?):(6553[0-5]|655[0-2][0-9]|65[0-4](\d){2}|6[0-4](\d){3}|[1-5](\d){4}|[1-9](\d){0,3})([,]){0,1})*/u</Mask>
```

Comment:
it requires simplifying the regex which is the original motivation :)

Reply:
Simplified it, not a whole lot shorter. The other option is a custom field type, though this works just fine